fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/rdf/RdfStream.java

```diff
@@ -13,12 +13,23 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+package org.fcrepo.kernel.api.rdf;
```

Comment:
Using the `org.fcrepo.kernel.api` package for `RdfStream` makes sense. What about the other Classes/Interfaces in the `api.rdf` package? Would you recommend putting all of those into the `org.fcrepo.kernel.api` package? (That would include `DefaultRdfStream`, `RdfContext` and `RdfCollectors`.